pom.xml

```diff
@@ -786,7 +786,7 @@
                       <ignoreClass>org.hornetq.utils.HornetQUtilBundle_$bundle</ignoreClass>
                       <ignoreClass>org.hornetq.utils.HornetQUtilLogger_$logger</ignoreClass>
                       <!-- Classes bundled in errai-javax-enterprise which are GWT friendly.
-                           See https://github.com/errai/errai/blob/master/errai-javax-enterprise/README -->
+                           See https://github.com/errai/errai/blob/main/errai-javax-enterprise/README -->
```

Comment:
are we going to rename errai repository default branch? :thinking:

Reply:
I see it will https://gist.github.com/mbiarnes/8987895c53e6ee352ba4ccdb78cab439
sorry @mbiarnes

Reply:
are we not? 
Will speak to Tiago.

Reply:
@mbiarnes as I told you on the internal chat it is my fault. sorry